lib/cretonne/src/isa/mod.rs

```diff
@@ -54,6 +54,8 @@ use result;
 use timing;
 use isa::enc_tables::Encodings;
 
+use std::boxed::Box;
```

Comment:
Whitespace nit: Various places like this still have a blank line between the `use` lines.

Reply:
Sorry about that. I'll remove those.